### ISSUE TITLE
no content here yet?

### ISSUE THREAD

**Issue by optimys** [WonderkidStudios/myhealthtest#24]
http://www.bugherd.com/projects/78317/tasks/9

**Comment by optimys** [comment]
You need exactly the same view as in results letter.doc for case when user has just one HbA1c result ?

**Comment by optimys** [comment]
I forgot to say.
User with one HbA1c result one_test@compass2k.com
health2015

**Comment by optimys** [comment]
yes.  the graph will just have a single point, and the block of text is ALWAYS required

**Comment by optimys** [comment]
Please confirm one Complete Basic layout for printing.
I'm starting to get confused

**Comment by optimys** [comment]
refer to results lyout RP.docx attached

**Comment by optimys** [comment]
The site is down from me but a global monitor has it up so looks like I have network problems - anyone else ?

**Comment by optimys** [comment]
TASK UPDATE

All working for me



*Marianne Gould*  BVSc MBA |  Marketing Director  |  *MyHealthTest Pty Ltd*

M: +61 417 473 602  |  E: marianne.gould@MyHealthTest.com  |  W:
www.myhealthtest.com



*From:* BugHerd [mailto:bug-Tj5BMfAjPXl8CxQ7prXIXw@bugs.bugherd.com]
*Sent:* Wednesday, 21 October 2015 1:25 PM
*To:* Marianne Gould <marianne.gould@myhealthtest.com>
*Subject:* Bug #9 has been updated (Health)

**Comment by compass2k** [comment]
You can process users etc ?  I'm getting timeouts from ftp as well as http
- and the rest of the web is good for me.

Will check with hostek what they are up to.

d

**Comment by optimys** [comment]
I purged my network cache and it came back - go figure.

**Comment by optimys** [comment]
I know what it was Alexey was having ftp issues a few days ago and had to reconnect a lot . So the IPS thought someone was trying to force ftp and flagged the machine. When I had  a couple of failures today it blocked my IP.
I sent the ISP a sour note asking them to check their ftp servers performance before locking out the already unfortunate would be users.

**Comment by optimys** [comment]
Marianne wrote "refer to results lyout RP.docx attached"
And Daniel wrote "..print view looks like in results-letter.doc "

So who's in charge?

**Comment by optimys** [comment]
We are all working to the same result. Marianne represents the business so knows more than me the latest specification.  Also her post is more recent and her document is more recent so hers is the latest specification.

**Comment by optimys** [comment]
Ok, but i have to rewrite code one more time.

**Comment by compass2k** [comment]
Sure -just put it as changes due respecification - but we are talking about
inserting blocks of static text and layout.

Plus be warned it will probably change again when Nick and Vicky look at it.

The interface changes are where most of the changes will be.

It wont be until we have a complete app that we will test the functionality
and performance
and then we will probably need to optimize and refactor.

But the user interface will probably be in flux forever.

d

**Comment by optimys** [comment]
Should i remove all other test? So i see that you don't need any backend logic for now?

**Comment by compass2k** [comment]
sure clean up anything uneccessary - call it  'codebase cleanup'.

d

**Comment by compass2k** [comment]
IT's 01:30 here so I will get some sleep and be back in 5 hours.

d

**Comment by compass2k** [comment]
Sorry I misunderstood you. I thought you were talking about cleaning up
leftover testing code from compass2k that had come across to
myhealthtestresults,com.

No - leave the other tests they are neccesary to demonstrate to the
business also we have to be able to handle multiple tests and complex
reports - even if some of the tests are dummies at present.

d

**Comment by compass2k** [comment]
myhealthtestresults.com is working with bugherd for me.
